### DISCUSSION TITLE
Support for offset reset with MM2.

### DISCUSSION THREAD

**Discussion by AAR256** [strimzi/strimzi-kafka-operator#9964] [Q&A]
Hi,

We're trying to setup MM2 with kafka version 3.4.0 and operator version 0.35.1. 

The use case I'm trying to implement is setting up MM2 and replicating only latest offsets as part of the setup. I can also see the feature has been described here in kafka repo - https://github.com/apache/kafka/pull/14567.

I'm trying to enable by applying this in my MM2 config - ```consumer.auto.offset.reset: 'latest'``` but it doesn't seems to be working. 

Can you confirm if this feature is supported with kafka/MM2 version 3.4.0. If not, Can you please help us identify which operator/kafka version has this feature enabled?

**Comment by scholzj** [comment]
That is a question on the Kafka project. You should probably find the related JIRA that will list the Kafka version where it was done.
